create-doks-with-terraform-flux/backend.tf

```diff
@@ -0,0 +1,13 @@
+# Store the state file using DO Spaces
+# DO Spaces is a S3-compatible object storage so the meaning of the properties used down below is the same
+
+terraform {
+  backend "s3" {
+    skip_credentials_validation = true
+    skip_metadata_api_check     = true
+    endpoint                    = "https://fra1.digitaloceanspaces.com" # replace this with your Spaces endpoint
+    region                      = "us-east-1"         # usually this can be left as it is
+    bucket                      = "doks-fluxcd"       # replace this with your bucket name
```

Comment:
The spaces bucket is called `tf-flux` in the README and I think it's easier if the bucket name here is the same.

Reply:
Nice catch. Fixed in the latest commit.